k8s/cloud_deps/public/postgres/postgres_deployment.yaml

```diff
@@ -4,11 +4,16 @@ kind: Deployment
 metadata:
   name: postgres
 spec:
+  strategy:
+    type: Recreate
   template:
     metadata:
       labels:
         name: postgres
     spec:
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
       containers:
       - name: postgres
         image: postgres:13.3
```

Comment:
@vihangm we use the debian based version here right? I set the `runAsUser: 999` and replaced the container command with a `sleep 3600` and SSH into the pod and checked what's going on.

<img width="533" alt="Screenshot 2023-02-06 at 15 27 55" src="https://user-images.githubusercontent.com/29277992/216941965-e0ca7327-288b-4dbb-a0b4-a99230e7fd41.png">

 Turns out `/var/lib/postgresql` is owned by root even though it was `postgres` user was given the ownership of this according to https://github.com/docker-library/postgres/blob/186c93e85d4c4fcee8c300fdfd2e9991c5d3efc9/13/bullseye/Dockerfile#L27. So it fails at https://github.com/docker-library/postgres/blob/master/13/bullseye/docker-entrypoint.sh#L41

Reply:
I did a bash trace and found out it actually fails at https://github.com/docker-library/postgres/blob/master/13/bullseye/docker-entrypoint.sh#L92

![Screenshot 2023-02-06 at 16 16 06](https://user-images.githubusercontent.com/29277992/216952074-4a1970c8-c1dd-4ae2-b67b-aad5d2ba939e.png)